src/Status/Parsing.elm

```diff
@@ -0,0 +1,8 @@
+module Status.Parsing exposing (statusParser)
+
+import UrlParser exposing (Parser, s, string, (</>))
```

Comment:
I'm super guilty of this (and I don't think it's actually a bad thing in this case), but you're exposing `string` and `(</>)` to the top-level space and not using them. The compiler so far has always informed me when I'm calling a function from the top level and it's not sure if I'm referencing `Foo.doSomething` or `Bar.doSomething`, so I'm not too concerned.